site/src/ThemeSelector.css

```diff
@@ -1,7 +1,4 @@
 .theme-selector {
-  position: fixed;
```

Comment:
Extremely nice work getting the two controls to flow/play well with one another, alignment-wise!